assets/index.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable global-require */
+export const animations = {
+  buttonLoading: require('./animations/buttonLoading.json'),
```

Comment:
I don't see this animation being tied anywhere directly to a button. I know that is the intended use case, but overall this is "just" a loading animation and it could possibly be used with something else.
Would prefer if it was just called `loading`

Reply:
I've (shamelessly :D) stole your LoadingButton code and removed button logic. Guess who forgot to remove/rename this one :D